web/vital_records/views.py

```diff
@@ -80,31 +86,33 @@ class StatementView(EligibilityMixin, ValidateRequestIdMixin, UpdateView):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
-        context["page_title"] = "Replacement birth records"
-
+        context["page_title"] = f"Replacement {self.object.type} records"
```

Comment:
This basically completes https://github.com/Office-of-Digital-Services/cdt-ods-disaster-recovery/issues/285 right?